03_homework/homework.sh

```diff
@@ -7,21 +7,35 @@
 # On your terminal, input all the commands you have used to create the following:
 
 # 1. How would you create 5 directories? Feel free to use any name for your directories.
-
+mkdir dir1 dir2 dir3 dir4 dir5
 # 2. How would you verify the creation of all 5 directories?
-
+ls
 # 3. In each directory, how would you create 5 .txt files and write "I love data" into each within the directories?
+touch file1.txt file2.txt file3.txt file4.txt file5.txt dir1
+code file1.text
+I love data
+cp -r dir1 dir2/
+cp -r dir1 dir3/
+cp -r dir1 dir4/
+cp -r dir1 dir5/
```

Comment:
`code file1.text
I love data
` Won't work as intended because you are calling on file1.text instead of file1.txt and "I love data" is not a command.

```
cp -r dir1 dir2/
cp -r dir1 dir3/
cp -r dir1 dir4/
cp -r dir1 dir5/
```
you are copying empty dir1 into other directory because `touch file1.txt file2.txt file3.txt file4.txt file5.txt dir1` doesn't create these files within dir1.